client/ayon_core/pipeline/editorial.py

```diff
@@ -179,6 +179,24 @@ def get_media_range_with_retimes(otio_clip, handle_start, handle_end):
     media_in = available_range.start_time.value
     media_out = available_range.end_time_inclusive().value
 
+    # Ensure image sequence media_ref source and
+    # available range are absolute.
+    media_ref = otio_clip.media_reference
+    if (
+        hasattr(otio.schema, "ImageSequenceReference") and
+        isinstance(media_ref, otio.schema.ImageSequenceReference) and
+        media_in != media_ref.start_frame
+    ):
+        media_in = media_ref.start_frame
+        media_out += media_ref.start_frame
+        source_range_start = otio.opentime.RationalTime(
+            media_in + source_range.start_time.value,
+        )
+        source_range = otio.opentime.TimeRange(
+            start_time=source_range_start,
+            duration=source_range.duration
+        )
+
```

Comment:
We need to test this with actual case data, as some image sequences may start at frame 1001, while the metadata on the frame could show `00:20:36:12`. What would be the outcome in that scenario?

Reply:
I'm gonna give it a try !

Reply:
Added a unit tests and logic with image sequence with embedded TC.